docs/api/multipart.rst

```diff
@@ -0,0 +1,51 @@
+.. _multipart:
+
+Multipart Forms
+===============
+
+Falcon features easy and efficient access to submitted multipart forms by using
+:class:`falcon.media.MultipartFormHandler` to handle the `multipart/form-data`
+:ref:`media <media>` type. ``req.media`` can then be used as an iterable
+through multipart form
+:class:`body parts <falcon.media.multipart.BodyPart>`:
+
+.. code:: python
+
+    for part in req.media:
+        if part.content_type == 'application/json':
+            # TODO: Body part is a JSON document, do something useful with it
+            resp.media = part.media
+        elif part.name = 'datafile':
+            while True:
+                # Do something with the uploaded data (file)
+                chunk = part.stream.read(8192)
+                if not chunk:
+                    break
+                feed_data(chunk)
+        else:
+            # TODO: Do something else
+
+.. note::
+   In further development versions of Falcon 3.0 series,
+   :class:`falcon.media.MultpartFormHandler` may be promoted to the default
```

Comment:
Probably a good idea, but I guess we can wait for community feedback.

Reply:
Do you suggest removing this paragraph?

Reply:
TBH I think we should just decide one way or another and do it if possible for the alpha. What are the potential problems with adding it to the default handlers dict?

Reply:
(If it is enabled by default, the docs should mention this fact.)

Reply:
Sure, let's do that :+1: 
There is one potential problem with circular imports, but I'll tape it one or another way,